modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +87,33 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+[discrete]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
+
+The Compatibility Support Module (CSM) configuration provides the support for legacy BIOS backward compatibility with UEFI systems.
+
+[NOTE]
+====
+When you are deploying with iRMC, you must configure the CSM, otherwise the cluster installation might fail.
+====
+
+[IMPORTANT]
+====
+Before you configure the CSM, ensure that you have disabled the `Secure Boot Control`, which is available under `Security` -> `Secure Boot Configuration`.
+====
+
+To configure the CSM, refer to the hardware guide for the node and execute the following:
```

Comment:
What specifically is required from the hardware guide that is not in this procedure? If there is a specific step that requires the hardware guide, you could state it explicitly as a step in this procedure.

You could also provide a link to the referenced hardware guide, so that the reader can easily refer to it. I guess if the guide is dependent on the type of hardware in each use case, this would not be possible. In that case, you could potentially link to a reference site that provides an index or search facility to find the relevant guide.

Reply:
If there is nothing in addition to the subsequent steps that are required from the guide, you could change this to a note.

Reply:
> What specifically is required from the hardware guide that is not in this procedure? If there is a specific step that requires the hardware guide, you could state it explicitly as a step in this procedure.

> If there is nothing in addition to the subsequent steps that are required from the guide, you could change this to a note.

Actually I referred to the writing method of the previous section "Manually configure the node for secure boot".  

No, no other steps are required for this line. Here is just to illustrate that the user needs to refer to the hardware guide to enter the BIOS menu and configure the CSM according to the following steps. But if change it to a note, will it be out of touch with the following content?

> You could also provide a link to the referenced hardware guide, so that the reader can easily refer to it. 

This section is specific to Fujitsu iRMC, so readers should have a hardware guide before deploying a cluster with iRMC. ​If the link is needed in this document, I will provide.

> I guess if the guide is dependent on the type of hardware in each use case, this would not be possible. In that case, you could potentially link to a reference site that provides an index or search facility to find the relevant guide.

Yes, the guide is dependent on the type of hardware in each use case and all the hardware guide can be found from https://support.ts.fujitsu.com

Reply:
> If there is nothing in addition to the subsequent steps that are required from the guide, you could change this to a note.

I change this to a note. PTAL.

Reply:
Thank you for addressing this suggestion. Now that it is a note in the introduction, I realise that in the note, readers are pointed to the hardware guide to configure the CSM and then we provide steps directly after the note on how to do exactly that. You could update the note to compliment the steps instead. For example:

> **Note**: For information about configuring the CSM for your specific node type, refer to the hardware guide for the node.

Reply:
+1